python/dgl/graphbolt/graph_storage/csc_sampling_graph.py

```diff
@@ -222,7 +222,53 @@ def sample_neighbors(
         # Ensure nodes is 1-D tensor.
         assert nodes.dim() == 1, "Nodes should be 1-D tensor."
         assert fanout >= 0 or fanout == -1, "Fanout shoud have value >= 0 or -1"
-        return self._c_csc_graph.sample_neighbors(nodes, fanout, replace)
+        return self._c_csc_graph.sample_neighbors(nodes, [fanout], replace)
+
+    def sample_etype_neighbors(
+        self,
+        nodes: torch.Tensor,
+        fanouts: torch.Tensor,
+        replace: bool = False,
+    ) -> torch.ScriptObject:
+        """Sample neighboring edges of the given nodes and return the induced
+        subgraph.
+
+        Parameters
+        ----------
+        nodes: torch.Tensor
+            IDs of the given seed nodes.
+        fanouts: torch.Tensor
```

Comment:
Update this according to the comment in c++ code.

Reply:
Fixed